packages/devtools/lib/src/framework/framework.dart

```diff
@@ -571,12 +659,58 @@ class ConnectDialog {
 
       // Hide the dialog
       hide();
+
+      // Hide the import message.
+      framework.importMessage.hide();
     } else {
       throw 'not connected';
     }
   }
 }
 
+class ImportMessage {
+  ImportMessage(this.framework) {
+    parent = CoreElement.from(queryId('import-message'));
+    parent.layoutVertical();
+
+    parent.add([
+      h2(text: 'Load DevTools Snapshot'),
+      CoreElement('dl', classes: 'form-group')
+        ..add([
+          CoreElement('dt')
+            ..add([label(text: 'Load a DevTools snapshot from a local file.')]),
+          CoreElement('dd')
+            ..add([
+              p(text: 'Drag and drop a file anywhere on the page.', c: 'note'),
+            ]),
+          CoreElement('dd')
+            ..add([
+              p(
+                  text: 'Supported file formats include any files exported from'
```

Comment:
we might want to also support generic chrome:trace files converting them over to work alright on the timeline view. That could help developers who happened to record a file in another tool.